### PR TITLE
slack email app

### PR DESCRIPTION
**Pull request by henrikskog**
related https://github.com/dotkom/terraform-monorepo/pull/145

### CODE REVIEW

File: apps/email-slack-app/src/core/gmail/InboxMethods.interface.ts
Comment (junlarsen) [inline apps/email-slack-app/src/core/gmail/InboxMethods.interface.ts:1]: Is there a reason we have .interface.ts in the file name? Should maybe these types just be a types.ts (or some other relevantly named file)?

File: apps/email-slack-app/src/lambda.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import * as Sentry from "@sentry/aws-serverless"
+import { nodeProfilingIntegration } from "@sentry/profiling-node"
+import type { APIGatewayProxyEventV2, APIGatewayProxyResultV2, Handler } from "aws-lambda"
+import { createExternalClients, createServiceLayer } from "./core"
+import { env } from "./env"
+
+Sentry.init({
+  dsn: "https://a07a847a3bbab26aac0ea417edcbc7f2@o4504113989287936.ingest.us.sentry.io/4508080738467840",
```
Comment (junlarsen) [inline apps/email-slack-app/src/lambda.ts:8]: Use an environment variable for this, and disable it in process.env.NODE_ENV !== production

File: apps/email-slack-app/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../packages/tsconfig/tsconfig.json",
+  "compilerOptions": {
+    "jsx": "react-jsx"
```
Comment (junlarsen) [inline apps/email-slack-app/tsconfig.json:4]: there's no jsx here right?

**Review by junlarsen — COMMENTED** [review]
Cool stuff, I suggest we find a more interesting name than email-slack-app (sorry if that causes a lot of terraform updates lol)

Other than that, looks pretty fun

File: apps/email-slack-app/src/core/gmail/InboxMethods.interface.ts
Comment (henrikskog) [reply apps/email-slack-app/src/core/gmail/InboxMethods.interface.ts:1]: yeah I agree it's weird, I stripped the code in /gmail from https://github.com/ismail-codinglab/gmail-inbox.

Comment (henrikskog) [reply apps/email-slack-app/src/core/gmail/InboxMethods.interface.ts:1]: I'll clean up stuff!

File: apps/email-slack-app/tsconfig.json (hunk 2 above)
Comment (henrikskog) [reply apps/email-slack-app/tsconfig.json:4]: Nope thanks

File: apps/email-slack-app/src/lambda.ts (hunk 1 above)
Comment (henrikskog) [reply apps/email-slack-app/src/lambda.ts:8]: Yeah good idea, thanks

**Pull request closed without merging**